examples/tree_2d_dgsem/elixir_advection_restart.jl

```diff
@@ -29,7 +29,7 @@ save_solution.condition.save_initial_solution = false
 
 integrator = init(ode, alg,
                   dt = dt, # solve needs some value here but it will be overwritten by the stepsize_callback
-                  save_everystep = false, callback = callbacks; ode_default_options()...)
+                  callback = callbacks, maxiters = 100_000; ode_default_options()...)
```

Comment:
We could also add `maxiters = 100_000` to `ode_default_options`, since `trixi_include` always adds this by default if the elixir doesn't contain `maxiters` already. AFAIK, this is also the default in OrdinaryDiffEq (which is why we set it to that value in `trixi_include` in the first place).

Reply:
Did you check whether `trixi_include` can overwrite stuff from `ode_default_options`? I would be a bit surprised since it doesn't appear explicitly in the source code itself...

Reply:
Ah right, I didn't think this through...

Reply:
Okay, then this won't work and we need the explicit settings as discussed in another comment 👍